engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/StorageSystemDataMotionStrategy.java

```diff
@@ -131,21 +132,23 @@
 
 @Component
 public class StorageSystemDataMotionStrategy implements DataMotionStrategy {
-    private static final Logger LOGGER = Logger.getLogger(StorageSystemDataMotionStrategy.class);
+    protected static final Logger LOGGER = Logger.getLogger(StorageSystemDataMotionStrategy.class);
```

Comment:
Why make this protected?

Reply:
@rhtyd I am using this logger on the `KvmNonManagedStorageDataMotionStrategy` class, thus have set as protected.
However, I shall fix it for `protected Logger logger = Logger.getLogger(getClass());`

Thanks for pointing that!

Reply:
My mistake here, @rhtyd. The class that extends this one does not need to log for now; thus, I kept this as private as it was before.